great_expectations/expectations/core/expect_column_values_to_be_dateutil_parseable.py

```diff
@@ -81,9 +82,25 @@ class ExpectColumnValuesToBeDateutilParseable(ColumnMapExpectation):
     }
     args_keys = ("column",)
 
+    @public_api
     def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
+        """Validates the configuration of an Expectation.
+
+        The configuration will also be validated using each of the `validate_configuration` methods in its Expectation
+        superclass hierarchy.
+
+        Args:
+            configuration: An `ExpectationConfiguration` to validate. If no configuration is provided, it will be pulled
+                           from the configuration attribute of the Expectation instance.
+
+        Raises:
+            InvalidExpectationConfigurationError: The configuration does not contain the values required by the
+                                                  Expectation.
+
+            # noqa: DAR402 InvalidExpectationConfigurationError
```

Comment:
Looks great, but I'm not sure if the `noqa` should be included here. Could I double check my understanding with @anthonyburdi ?

Reply:
I decided to ignore the linting error since the `super` calls raise this error, and if any actual logic gets added to this method it should also raise this error.

Reply:
You can remove this noqa, the new linter does not support this error. We may add it back in the future but you can take it out for now.

Reply:
```suggestion
```

Reply:
But yeah otherwise looks good to me!